src/Symfony/Component/Finder/Finder.php

```diff
@@ -142,7 +142,72 @@ public function depth($level)
      */
     public function date($date)
     {
-        $this->dates[] = new Comparator\DateComparator($date);
+        $this->dates[] = new Comparator\DateComparator($date, Comparator\DateComparator::TIME_TYPE_MODIFIED);
+
+        return $this;
+    }
+
+    /**
+     * Adds tests for file dates (last accessed).
+     *
+     * Accessed date is the last time the file was:
+     *
+     *   read
+     *   written
+     *   permissions changed
+     *   moved
+     *   renamed
+     *
+     * The date must be something that strtotime() is able to parse:
+     *
+     *   $finder->dateAccessed('since yesterday');
+     *   $finder->dateAccessed('until 2 days ago');
+     *   $finder->dateAccessed('> now - 2 hours');
+     *   $finder->dateAccessed('>= 2005-10-15');
+     *
+     * @param string $date A date range string
+     *
+     * @return $this
+     *
+     * @see strtotime
+     * @see DateRangeFilterIterator
+     * @see DateComparator
+     */
+    public function dateAccessed($date)
```

Comment:
Instead of create new methods, what about adding an argument to the `date` one?

Reply:
I've added an argument to date method and called the same method from these.
I feel that it's clearer to the user to have separate methods with documentation to describe what the difference is between each method.

Reply:
I would remove the other methods as the class already have so many methods.